setup.py

```diff
@@ -22,6 +22,10 @@
 SRC_DIR = os.path.join(TOP_DIR, 'onnx')
 TP_DIR = os.path.join(TOP_DIR, 'third_party')
 PROTOC = find_executable('protoc')
+ONNX_ML=False
+if "--onnxml=1" in sys.argv:
```

Comment:
Use user_options to add this flag.

Reply:
I'm using --install-option="--onnxml=1". 
"user_options", you mean options for a command, right? I tried that and it needs to send the flag around, and I'd use it as a global setting. I'll keep it as it is if there's no strong concern. Thank you!

Reply:
This seems really questionable. Let's at least document it as such. You probably got this from the second answer to the top listed SO question on this subject?

Reply:
OK. let me document it. I also don't like two ways of building onnx. However, as long as we have onnx-ml and onnx separately, it has to be there, right? :).

Reply:
@linkerzhang Yeah I think @ezyang is not questioning about adding a flag, but the way you do command line parsing here (e.g. it won't show up in the help message, user can not do `--onnxml` or `--onnxml 1`).